src/SFA.DAS.AssessorService.Web/Views/Dashboard/Index.cshtml

```diff
@@ -10,6 +10,10 @@
 <main class="govuk-main-wrapper " id="main-content" role="main">
     <div class="govuk-grid-row">
 
+        <h1 class="govuk-heading-xl">
+            Apprenticeship assessment service
```

Comment:
Did this come from Content or UX design - I'm surprised to see this

Reply:
Need to talk with Victor about a screen design change only for Accessibility which seems wrong and to James to see if the same text as this which is in the header bar could become the h1 which is needed for the accessibility

Reply:
Update with new hidden content per conversation with James King, Jazzy and Victor